src/server/pps/pretty/pretty.go

```diff
@@ -57,7 +57,7 @@ type KubeEvent struct {
 		Object   struct {
 			Name string `json:"name"`
 		} `json:"involvedObject"`
-	} `json:"event"`
+	} `json:"kube-event"`
```

Comment:
Why not keep the name `event`? Also, it looks like at least the message above is named with `camelCase` rather than `kebab-case`.

I prefer `kebab-case`, but consistency is probably best.